apps/bare-expo/ios/Podfile

```diff
@@ -61,6 +61,7 @@ target 'BareExpo' do
     }
   )
 
+  pod 'EXDevMenu', path: '../../../packages/expo-dev-menu'
```

Comment:
Any reason to not use auto-linking for this? (I'm using it in 'expo-development-client')

Reply:
Are you sure RN autolinking works for packages with custom paths like this? I think we call `use_native_modules!` only to link packages that are not within the workspace and installed under `node_modules`.